### DISCUSSION TITLE
영상처리 문제 런타임 에러

### DISCUSSION THREAD

**Discussion by DWL21** [woowacourse-study/2022-lv3-algorithm-study#129] [정답 원해요...] [answered]
파이참에서 예시 케이스는 돌아가는데 ValueError가 뜹니다ㅠㅠ

https://github.com/woowacourse-study/2022-lv3-algorithm-study/pull/128/commits/3ecb72f22b8f84bcfa2979f30e967907874039ef

<img width="1178" alt="스크린샷 2022-07-23 17 49 21" src="https://user-images.githubusercontent.com/76774809/180597897-7a388bae-086f-4d61-b9f5-1f1546759996.png">

**Comment by sinb57** [comment, marked as answer]
입력 예시
```
1 2
1 1 1 0 0 0
1
```
---

출력 예시
```
1
```

**Reply by DWL21** [reply]
가로,세로를 헷갈렸네요. 해결했습니다! 감사합니다 이프
